it/_mixins.md

```diff
@@ -17,10 +17,7 @@ Un altro esempio valido potrebbe essere un _Mixin_ per impostare la grandezza di
 
 {% include snippets/mixins/02/index.html %}
 
-###### Approfondimenti
-
-* [Sass Mixins to Kickstart your Project](http://www.sitepoint.com/sass-mixins-kickstart-project/)
-* [A Sass Mixin for CSS Triangles](http://www.sitepoint.com/sass-mixin-css-triangles/)
+For more complex examples of mixins, have a look at [this mixin to generate CSS triangles](http://www.sitepoint.com/sass-mixin-css-triangles/), [this mixin to create long shadows](http://www.sitepoint.com/ultimate-long-shadow-sass-mixin/) or [this mixin to polyfill CSS gradients for old browsers](http://www.sitepoint.com/building-linear-gradient-mixin-sass/).
```

Comment:
This sentence was missing. I added it in English, but it would need to be translated. :)